pkg/cli/images_all_test.go

```diff
@@ -0,0 +1,246 @@
+package cli
+
+import (
+	"context"
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	apiv1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/acorn-io/acorn/pkg/mocks"
+	"github.com/acorn-io/acorn/pkg/project"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/acorn-io/acorn/pkg/cli/testdata"
+	"github.com/golang/mock/gomock"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestImages(t *testing.T) {
+	image1Untagged := apiv1.Image{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "project1::image-1-9102390",
+			Namespace: "NameOne",
+		},
+		Digest: "1234567890asdfghkl",
+	}
+	image1Tagged := image1Untagged
+	image1Tagged.Tags = []string{"testtag:latest"}
+
+	image2Untagged := apiv1.Image{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "project2::image-2-2198031",
+			Namespace: "NameTwo",
+		},
+	}
+
+	image2Tagged := image2Untagged
+	image2Tagged.Tags = []string{"testtag2:latest2"}
+
+	image3Untagged := apiv1.Image{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "acorn.com/acc/Name3::image-3-129394",
```

Comment:
I'd suggest using `example.com` here. I know that in simple testing like this, nothing is going to try to reach out to that domain, but it's probably be best to avoid using something like that that we don't own. (Not that we own example.com, but it's meant to be used as an example of course.)

Reply:
Done, fun logic :)